cloud/k8s/deploy/kontain-deploy/overlays/kkm/cm-install-lib-kkm.yaml

```diff
@@ -56,5 +64,10 @@ data:
       chroot ${ROOT_MOUNT_DIR} ln -sf /opt/kontain/shim/containerd-shim-krun-v2 /usr/bin/containerd-shim-krun-v2
 
       chmod +x ${ROOT_MOUNT_DIR}/kontain-artifacts/kkm.run
-      chroot ${ROOT_MOUNT_DIR}   /kontain-artifacts/kkm.run -- --force-install
+
+      sleep 2
```

Comment:
This was necessary because when testing the install on worker nodes on AWS that were smaller, there was an issue with some CPU flags, and talking with Sreeni I had to use the --force-install.  Also, the sleep was necessary on slower systems after package installs/updates else it wouldn't work.  This was most apparent on Vagrant systems like on Mac OS which used sshfs for vmware or virtualbox based installs and had slow disks.

Reply:
No issues with `--force-install`. My point is that `sleep 2` doesn't guarantee that the state is correct. We need to find out what to wait for to make sure the state is correct?